pkg/manager/impl/execution_manager_test.go

```diff
@@ -2818,7 +2818,7 @@ func TestTerminateExecution(t *testing.T) {
 		}, unmarshaledClosure.GetAbortMetadata()))
 		return nil
 	}
-	repository.ExecutionRepo().(*repositoryMocks.MockExecutionRepo).SetUpdateExecutionCallback(updateExecutionFunc)
+	repository.ExecutionRepo().(*repositoryMocks.MockExecutionRepo).SetUpdateCallback(updateExecutionFunc)
```

Comment:
thanks for simplifying these, we should really move to using mockery so these don't have to be handrolled (https://github.com/flyteorg/flyte/issues/149)